functions/src/import/index.ts

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2022 Meilisearch
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as admin from 'firebase-admin'
+import { CliConfig, parseConfig } from './config'
+import * as logs from '../logs'
+import { adaptDocument } from '../adapter'
+import { initMeilisearchIndex } from '../meilisearch/create-index'
+import { Index } from '../types'
+import { DocumentSnapshot } from 'firebase-functions/lib/providers/firestore'
+
+const run = async () => {
+  // Retrieve all arguments from the commande line.
+  const config: CliConfig = await parseConfig()
+
+  // Initialize Firebase using the Google Credentials in the GOOGLE_APPLICATION_CREDENTIALS environment variable.
+  admin.initializeApp({
+    credential: admin.credential.applicationDefault(),
```

Comment:
If these are customizable by the user they are not `default`. Default would mean that there is a value that the user does not have to communicate, like `http://localhost:7700` for meilisearch when launching the server.

Reply:
I'm not sure to understand what you suggest. This function is a `firebase` tools for administrating your app:
https://firebase.google.com/docs/reference/admin/node/firebase-admin.app.md#applicationdefault